README.nfsroot.md

```diff
@@ -15,25 +15,17 @@ Build your rootfs recipe:
 bitbake my-rootfs-image
 ```
 
-Export your rootfs:
-
-```
-NFSROOT=$BUILDDIR/my-nfsroot
-bitbake qemu-helper-native -caddto_recipe_sysroot
```

Comment:
After deleting my `$BUILDDIR/tmp` because of pseudo aborts, re-building my image and starting the nfs-export-updater again, I'm now running into:

```
INFO: Running ('runqemu-extract-sdk', '[...]/YOCTO.BSP-Foo/build/tmp/deploy/images/my-machine/my-image-my-machine.tar.gz', '[...]/YOCTO.BSP-Foo/build/nfsroot-my-image-my-machine')...
INFO:nfs-export-updater:Running ('runqemu-extract-sdk', '[...]/YOCTO.BSP-Foo/build/tmp/deploy/images/my-machine/my-image-my-machine.tar.gz', '[...]/YOCTO.BSP-Foo/build/nfsroot-my-image-my-machine')...
Running bitbake -e qemu-helper-native
Error: [...]/YOCTO.BSP-Foo/build/tmp/work/x86_64-linux/qemu-helper-native/1.0-r1/recipe-sysroot-native doesn't exist.
Have you run 'bitbake qemu-helper-native -caddto_recipe_sysroot'?
Traceback (most recent call last):
  File "[...]/YOCTO.BSP-Foo/poky/scripts/nfs-export-updater", line 166, in <module>
    main()
  File "[...]/YOCTO.BSP-Foo/poky/scripts/nfs-export-updater", line 135, in main
    create_nfs_exportdir(args.rootfs, args.exportdir)
  File "[...]/YOCTO.BSP-Foo/poky/scripts/nfs-export-updater", line 54, in create_nfs_exportdir
    raise Exception('Failed to run %s' % str(cmd))
Exception: Failed to run ('runqemu-extract-sdk', '[...]/YOCTO.BSP-Foo/build/tmp/deploy/images/my-machine/my-image-my-machine.tar.gz', '[...]/YOCTO.BSP-Foo/build/nfsroot-my-image-my-machine')
```

Reply:
Oh, yes. We need to invoke that task manually, too now.